ovs/proto_trace.go

```diff
@@ -35,6 +35,7 @@ var (
 	traceFlowRegexp       = regexp.MustCompile(` *[[:digit:]]+[.] ([[:alpha:]].*)`)
 	traceActionRegexp     = regexp.MustCompile(` +([[:alpha:]].*)`)
 	recircIDRegexp        = regexp.MustCompile(`recirc_id=`)
+	recircRegexp          = regexp.MustCompile(`recirc`)
```

Comment:
Could this get a little too greedy and match other lines with `recirc_id=...` or something else in it? Or, are you depending on matching order to prefer other matches for those lines?